conf/muchos.props.example

```diff
@@ -278,10 +278,11 @@ yarn_nm_mem_mb=16384
 # Where:
 #   Hostname = Must be unique.  Will be used for hostname in EC2 or should match hostname on your own cluster
 #   Service = Service to run on node (possible values: zookeeper, namenode, resourcemanager, accumulomaster, client, swarmmanager,
-#             mesosmaster, worker, fluo, metrics, spark). The following services are required: namenode, resourcemanager,
+#             mesosmaster, worker, fluo, metrics, spark, elkserver). The following services are required: namenode, resourcemanager,
```

Comment:
I think this mention of elkserver depends on PR #338, doesn't it?

Reply:
Yes, I will remove it from here and update #338 with that service.